hypha/apply/dashboard/templates/dashboard/applicant_dashboard.html

```diff
@@ -1,84 +1,137 @@
-            {% endfor %}
-        </div>
-    </div>
 
-    {% if active_projects.count %}
-        <div class="wrapper wrapper--large wrapper--inner-space-medium mb-8">
-            <h2 class="text-xl mb-2">{% trans "Your active projects" %}</h2>
-            {% render_table active_projects.table %}
-        </div>
-    {% endif %}
-
-    {% if historical_submissions.count %}
-        <div class="wrapper wrapper--large wrapper--inner-space-medium mb-8">
-            <h2 class="text-xl mb-2">{% trans "Submission history" %}</h3>
-            {% render_table historical_submissions.table %}
-        </div>
-    {% endif %}
-
-    {% if historical_projects.count %}
-        <div class="wrapper wrapper--large wrapper--inner-space-medium mb-8">
-            <h2 class="text-xl mb-2">{% trans "Project history" %}</h2>
-            {% render_table historical_projects.table %}
-        </div>
-    {% endif %}
+                <div class="mb-10">
+                    <div class="flex">
+                        <h2 class="font-light flex-1">{% trans "My active invoices" %}</h2>
+                        <p class="font-bold"><a href="{% url 'apply:projects:invoices' %}" target="_blank">{% trans "View all invoices" %}</a></p>
+                    </div>
+                    {% for invoice in active_invoices.data %}
+                        <div class="wrapper wrapper--status-bar-outer">
+                            <div class="wrapper wrapper--status-bar-inner pl-4 pr-4">
+                                <div class="max-w-[33%] w-full text-left my-auto">
+                                    <h4 class="heading heading--no-margin font-bold"><a class="link link--underlined" href="{{ invoice.get_absolute_url }}">
+                                        {% if invoice.invoice_number %}{{ invoice.invoice_number }}{% else %}{{ invoice.vendor_document_number }}{% endif %}
+                                    </a></h4>
+                                    <p class="m-0 text-gray-400">{% trans "Date added: " %} {{ invoice.requested_at }}</p>
+                                </div>
+                                <div class="max-w-[33%] w-full text-center my-auto">
+                                    <p class="text-2xl">{% if invoice.invoice_amount %}{{ invoice.invoice_amount | format_number_as_currency }}{% else %}-{% endif %}</p>
+                                </div>
+                                <div class="max-w-[33%] w-full flex my-auto">
+                                    {% display_invoice_status_for_user request.user invoice as invoice_status %}
+                                    <div class="w-full flex-1"></div>
+                                    <div class="max-w-fit w-full text-right">
+                                        <p class="bg-{{ invoice.get_status_display|invoice_status_colour }}-100 text-base py-2 px-3 text-{{ invoice.get_status_display|invoice_status_colour }}-600">{{ invoice_status }}</p>
```

Comment:
Using this will format will not generate the css classes required for the colors that are not know or used before. I would suggest creating a python dictionary with all the color values and then use them here. 

Then attach two funtions to the invoice model that gives you the color values. 

## sudo code

```
1. status and color mapping, use full class names. twailwind is configured to find classes in the python file as well. 

2. use `invoice.status_css_class_bg` and `invoice.status_css_class_fg` here.
```